LabTools/CoreWidgets/FunctionWidget.py

```diff
@@ -0,0 +1,995 @@
+# -*- coding: utf-8 -*-
```

Comment:
@zackorenberg - As this one is not working yet, I would copy it outside and add it in a subsequent PR. Would you agree

Reply:
It is working perfectly on my computer with Keithley2500. Currently that is the only driver that I have programmed it for.

Reply:
Oh, then you should remove the mention "Not working YET" from the module's docstring ^^. I could only see that the widget was loading, but not testing it with instrument unfortunately.

Reply:
done, will be pushed with my next commit